### PR TITLE
Documentation for Athena test setup

### PR DESCRIPTION
**Pull request by thijs-obj**
Add documentation for the athena test setup

Part 2/3 after splitting https://github.com/objectiv/objectiv-analytics/pull/1097

### CODE REVIEW

File: bach/tests/athena-setup-test-db.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+**Important** - All descriptions, code fragments, and configuration in this section assume that:
+1. The aws-account-id is `123456789012`
+2. The s3 staging bucket (to be created) is named `obj-automated-tests`
+
+When setting up a new Athena/S3 environment make sure to edit those values appropriately.
+
+TODO:
+* Add similar file for BigQuery
+* Improve this file
```
Comment (KathiaBarahona) [inline bach/tests/athena-setup-test-db.md:12]: maybe an issue for this?

File: bach/README.md [hunk 2]
```diff
@@ -30,19 +30,34 @@ pip install -e .[dev,bigquery]
...
+|  Athena  | `OBJ_DB_ATHENA_TEST_URL`               | Database URL                 |
+|  Athena  | `OBJ_DB_ATHENA_AWS_ACCESS_KEY_ID`      | Access key id*               |
+|  Athena  | `OBJ_DB_ATHENA_AWS_SECRET_ACCESS_KEY`  | Secret key*                  |
+|  Athena  | `OBJ_DB_ATHENA_REGION_NAME`            | Region*                      |
+|  Athena  | `OBJ_DB_ATHENA_SCHEMA_NAME`            | Schema name*                 |
+|  Athena  | `OBJ_DB_ATHENA_S3_STAGING_DIR`         | S3 staging dir*              |
+|  Athena  | `OBJ_DB_ATHENA_WORK_GROUP`             | Workgroup to run queries as* |
+|          |                                        |                              |
+| BigQuery | `OBJ_DB_BQ_TEST_URL`                   |                              |
+| BigQuery | `OBJ_DB_BQ_CREDENTIALS_PATH`           | Database URL                 |
```
Comment (KathiaBarahona) [inline bach/README.md:50]:
```suggestion
| BigQuery | `OBJ_DB_BQ_TEST_URL`                   | Database URL                 |
| BigQuery | `OBJ_DB_BQ_CREDENTIALS_PATH`           | Credentials Path             |
```

File: bach/README.md [hunk 3]
```diff
@@ -30,19 +30,34 @@ pip install -e .[dev,bigquery]
...
+
+For Athena one can either specify the full connection string as `OBJ_DB_ATHENA_TEST_URL`, or set
```
Comment (KathiaBarahona) [inline bach/README.md:52]: **nit-pick**
```suggestion
For Athena one can either specify the full connection string as `OBJ_DB_ATHENA_TEST_URL` or set
```

**Review by KathiaBarahona — APPROVED** [review]
lgtm overall

File: bach/README.md (hunk 3 above)
Comment (thijs-obj) [reply bach/README.md]: changed d497710173a4af0dfcaab4e92f3cf19180f1b58c

File: bach/README.md (hunk 2 above)
Comment (thijs-obj) [reply bach/README.md]: doh! fixed d497710173a4af0dfcaab4e92f3cf19180f1b58c

File: bach/tests/athena-setup-test-db.md (hunk 1 above)
Comment (thijs-obj) [reply bach/tests/athena-setup-test-db.md]: Removed this TODO in d497710173a4af0dfcaab4e92f3cf19180f1b58c and added issue https://github.com/objectiv/objectiv-analytics/issues/1114

**Pull request merged**